scripts/pylib/twister/twisterlib/twister_main.py

```diff
@@ -241,6 +241,8 @@ def main(options, default_options):
         or (tplan.warnings and options.warnings_as_errors)
         or (options.coverage and not coverage_completed)
     ):
+        if env.options.quit_on_failure:
+            logger.info("twister aborted because of a failure/error")
```

Comment:
now it says 'Run completed' above and then 'twister aborted...' that is confusing, it is either completed or aborted.

Reply:
I see, updated.